README.md

```diff
@@ -17,16 +17,28 @@ When one gives laptop to his/her aquinatance, he/she can start locking session a
 
   
  ### Prerequisites  
-       
+  ```sh
+       1. Android Studio
```

Comment:
is it necessary to have android studio??

Reply:
i think we can use any suitable editor like atom or any other.

Reply:
@akash1309 android studio is not required necessarily. But generally, for android projects, we use an android studio, which in turn installs android and way to do Gradle build.
if one uses atom or another, he/she have to configure multiple things on their on i think.

Reply:
i think it can be done by titanium build.But lets build it on android , it will be easy.